exporter/api.py

```diff
@@ -0,0 +1,74 @@
+"""
+Functions for exporting learning resources
+"""
+
+from __future__ import unicode_literals
+
+from tempfile import mkdtemp, mkstemp
+import tarfile
+import os
+from shutil import rmtree
+
+from django.core.files.storage import default_storage
+from lore.settings import EXPORT_PATH_PREFIX
+
+
+def export_resources_to_directory(learning_resources, tempdir):
+    """
+    Create XML files of learning resource contents inside directory.
+
+    Args:
+        learning_resources (list of learningresources.models.LearningResource):
+            LearningResources to export in tarball
+    """
+    def sanitize(title):
```

Comment:
We still need to truncate the filename below the file system limit. Does slugify do that?